From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Giuliano Procida' via kernel-team <kernel-team@android.com>
Date: Wed, 1 Jul 2020 11:54:03 +0100
Subject: abidiff: Fix report whitespace typos.

The patch fixes two minor formatting typos.

	* src/abg-reporter-priv.cc (represent): Add missing space to
	string split across two lines in certain anonymous data member
	diifs.
	* src/abg-default-reporter.cc (report): In the array_diff
	overload, eliminate trailing space at end of line.
	* tests/data/test-diff-dwarf/test10-report.txt: Delete
	trailing whitespace.
	* tests/data/test-diff-filter/test24-compatible-vars-report-1.txt:
	Ditto.
	* tests/data/test-diff-pkg/libICE-1.0.6-1.el6.x86_64.rpm--libICE-1.0.9-2.el7.x86_64.rpm-report-0.txt:
	Ditto.
	* tests/data/test-diff-pkg/libcdio-0.94-1.fc26.x86_64--libcdio-0.94-2.fc26.x86_64-report.1.txt:
	Ditto.
	* tests/data/test-diff-pkg/tbb-4.1-9.20130314.fc22.x86_64--tbb-4.3-3.20141204.fc23.x86_64-report-0.txt:
	Ditto.

Signed-off-by: Giuliano Procida <gprocida@google.com>
Signed-off-by: Matthias Maennich <maennich@google.com>
---
 src/abg-default-reporter.cc                                 | 2 +-
 src/abg-reporter-priv.cc                                    | 2 +-
 tests/data/test-diff-dwarf/test10-report.txt                | 2 +-
 .../test-diff-filter/test24-compatible-vars-report-1.txt    | 2 +-
 ...6.x86_64.rpm--libICE-1.0.9-2.el7.x86_64.rpm-report-0.txt | 2 +-
 ...4-1.fc26.x86_64--libcdio-0.94-2.fc26.x86_64-report.1.txt | 2 +-
 ...fc22.x86_64--tbb-4.3-3.20141204.fc23.x86_64-report-0.txt | 6 +++---
 7 files changed, 9 insertions(+), 9 deletions(-)

diff --git a/src/abg-default-reporter.cc b/src/abg-default-reporter.cc
index 2acb6954a22a..69941e5f6b94 100644
--- a/src/abg-default-reporter.cc
+++ b/src/abg-default-reporter.cc
@@ -632,7 +632,7 @@ default_reporter::report(const array_diff& d, ostream& out,
       string fn = ir::get_pretty_representation(is_type(dif->first_subject()));
       // report array element type changes
       out << indent << "array element type '"
-	  << fn << "' changed: \n";
+	  << fn << "' changed:\n";
       dif->report(out, indent + "  ");
     }
 
diff --git a/src/abg-reporter-priv.cc b/src/abg-reporter-priv.cc
index 2ca48c2143af..2e148fbb5b85 100644
--- a/src/abg-reporter-priv.cc
+++ b/src/abg-reporter-priv.cc
@@ -469,7 +469,7 @@ represent(const var_diff_sptr	&diff,
 	  out << indent << "while looking at anonymous data member '"
 	      << o_pretty_representation << "':\n"
 	      << indent << "the internal name of that anonymous data member"
-			   "changed from:\n"
+			   " changed from:\n"
 	      << indent << " " << get_type_name(o_type) << "\n"
 	      << indent << "to:\n"
 	      << indent << " " << get_type_name(n_type) << "\n"
diff --git a/tests/data/test-diff-dwarf/test10-report.txt b/tests/data/test-diff-dwarf/test10-report.txt
index ea561e89a599..52786b54d282 100644
--- a/tests/data/test-diff-dwarf/test10-report.txt
+++ b/tests/data/test-diff-dwarf/test10-report.txt
@@ -13,7 +13,7 @@ Variables changes summary: 0 Removed, 0 Changed, 0 Added variable
             array type size changed from 480 to 640
             array type subrange 2 changed length from 3 to 4
           type of 'int* S::m1[10]' changed:
-            array element type 'int*' changed: 
+            array element type 'int*' changed:
               in pointed to type 'int':
                 type name changed from 'int' to 'char'
                 type size changed from 32 to 8 (in bits)
diff --git a/tests/data/test-diff-filter/test24-compatible-vars-report-1.txt b/tests/data/test-diff-filter/test24-compatible-vars-report-1.txt
index d370eefd6b52..49e33b0b926f 100644
--- a/tests/data/test-diff-filter/test24-compatible-vars-report-1.txt
+++ b/tests/data/test-diff-filter/test24-compatible-vars-report-1.txt
@@ -5,7 +5,7 @@ Variables changes summary: 0 Removed, 1 Changed, 0 Added variable
 
   [C] 'S a[2]' was changed to 'typedef_of_S a[2]':
     type of variable changed:
-      array element type 'struct S' changed: 
+      array element type 'struct S' changed:
         entity changed from 'struct S' to compatible type 'typedef typedef_of_S'
       type name changed from 'S[2]' to 'typedef_of_S[2]'
       type size hasn't changed
diff --git a/tests/data/test-diff-pkg/libICE-1.0.6-1.el6.x86_64.rpm--libICE-1.0.9-2.el7.x86_64.rpm-report-0.txt b/tests/data/test-diff-pkg/libICE-1.0.6-1.el6.x86_64.rpm--libICE-1.0.9-2.el7.x86_64.rpm-report-0.txt
index 487c672ce2da..61b03335ba9f 100644
--- a/tests/data/test-diff-pkg/libICE-1.0.6-1.el6.x86_64.rpm--libICE-1.0.9-2.el7.x86_64.rpm-report-0.txt
+++ b/tests/data/test-diff-pkg/libICE-1.0.6-1.el6.x86_64.rpm--libICE-1.0.9-2.el7.x86_64.rpm-report-0.txt
@@ -86,7 +86,7 @@
 
     [C] 'char* _IceAuthNames[1]' was changed to 'const char* _IceAuthNames[1]' at globals.h:37:1:
       type of variable changed:
-        array element type 'char*' changed: 
+        array element type 'char*' changed:
           in pointed to type 'char':
             entity changed from 'char' to 'const char'
             type size hasn't changed
diff --git a/tests/data/test-diff-pkg/libcdio-0.94-1.fc26.x86_64--libcdio-0.94-2.fc26.x86_64-report.1.txt b/tests/data/test-diff-pkg/libcdio-0.94-1.fc26.x86_64--libcdio-0.94-2.fc26.x86_64-report.1.txt
index 929f43a393e7..cba145c5aa18 100644
--- a/tests/data/test-diff-pkg/libcdio-0.94-1.fc26.x86_64--libcdio-0.94-2.fc26.x86_64-report.1.txt
+++ b/tests/data/test-diff-pkg/libcdio-0.94-1.fc26.x86_64--libcdio-0.94-2.fc26.x86_64-report.1.txt
@@ -20,7 +20,7 @@
                   type size hasn't changed
                   1 data member change:
                     type of 'cdio_dvd_layer_t cdio_dvd_physical::layer[4]' changed:
-                      array element type 'typedef cdio_dvd_layer_t' changed: 
+                      array element type 'typedef cdio_dvd_layer_t' changed:
                         underlying type 'struct cdio_dvd_layer' at dvd.h:65:1 changed:
                           type size hasn't changed
                           8 data member changes:
diff --git a/tests/data/test-diff-pkg/tbb-4.1-9.20130314.fc22.x86_64--tbb-4.3-3.20141204.fc23.x86_64-report-0.txt b/tests/data/test-diff-pkg/tbb-4.1-9.20130314.fc22.x86_64--tbb-4.3-3.20141204.fc23.x86_64-report-0.txt
index 00e19379b88e..4fc20c36bce3 100644
--- a/tests/data/test-diff-pkg/tbb-4.1-9.20130314.fc22.x86_64--tbb-4.3-3.20141204.fc23.x86_64-report-0.txt
+++ b/tests/data/test-diff-pkg/tbb-4.1-9.20130314.fc22.x86_64--tbb-4.3-3.20141204.fc23.x86_64-report-0.txt
@@ -42,7 +42,7 @@
                         1 data member change:
                           'tbb::spin_mutex tbb::internal::concurrent_monitor::mutex_ec' offset changed from 0 to 8 (in bits) (by +8 bits)
                       type of 'tbb::internal::micro_queue tbb::internal::concurrent_queue_rep::array[8]' changed:
-                        array element type 'struct tbb::internal::micro_queue' changed: 
+                        array element type 'struct tbb::internal::micro_queue' changed:
                           type size hasn't changed
                           1 data member change:
                             type of 'tbb::spin_mutex tbb::internal::micro_queue::page_mutex' changed:
@@ -96,7 +96,7 @@
                             struct tbb::internal::padded<tbb::internal::arena_base, 128ul> at tbb_stddef.h:251:1
                           1 data member change:
                             type of 'tbb::internal::arena_slot tbb::internal::arena::my_slots[1]' changed:
-                              array element type 'struct tbb::internal::arena_slot' changed: 
+                              array element type 'struct tbb::internal::arena_slot' changed:
                                 type size hasn't changed
                                 2 base class deletions:
                                   struct tbb::internal::padded<tbb::internal::arena_slot_line1> at tbb_stddef.h:261:1
@@ -154,7 +154,7 @@
                         'intptr_t tbb::internal::market::my_global_bottom_priority' offset changed from 448 to 512 (in bits) (by +64 bits)
                         'uintptr_t tbb::internal::market::my_global_reload_epoch' offset changed from 512 to 576 (in bits) (by +64 bits)
                         type of 'tbb::internal::market::priority_level_info tbb::internal::market::my_priority_levels[3]' changed:
-                          array element type 'struct tbb::internal::market::priority_level_info' changed: 
+                          array element type 'struct tbb::internal::market::priority_level_info' changed:
                             type size hasn't changed
                             1 data member change:
                               type of 'tbb::internal::intrusive_list_base<tbb::internal::intrusive_list<tbb::internal::arena>, tbb::internal::arena>::iterator tbb::internal::market::priority_level_info::next_arena' changed:
